src/common/Accessibility.ts

```diff
@@ -264,11 +266,7 @@ class AXNode {
   private _isPlainTextField(): boolean {
     if (this._richlyEditable) return false;
     if (this._editable) return true;
-    return (
-      this._role === 'textbox' ||
-      this._role === 'ComboBox' ||
```

Comment:
Out of curiosity, what was this 'ComboBox' role and why is it not relevant anymore?

Reply:
I'm definitely now very knowledgeable in the area, but from searching around I concluded that there is no such role in the spec. There is a `combobox`, but I'm not sure we want to categorize those as plain text fields? At least our tests don't want that. I think we should remove the `ComboBox` for now as it is confusing.